File: Codeforces/Golang/800-1000/1030A - In Search of an Easy Problem/main.go
package main

import (
	"fmt"
	"slices"
)

func main() {
	var n int
	fmt.Scan(&n)

	arry := []int{}

	for i := 0; i < n; i++ {
		var j int
		fmt.Scan(&j)
		arry = append(arry, j)
	}
	oneOrZero := slices.Contains(arry, 1)

	if oneOrZero {
		fmt.Println("HARD")
	} else {
		fmt.Println("EASY")
	}
}

File: Codeforces/Rust/800-1000/1030A - In Search of an Easy Problem/src/main.rs
use std::io;

fn main() {
    let mut input = String::new();
    let _ = io::stdin().read_line(&mut input);

    let _n: i64 = input.trim().parse().expect("It's not number");

    input.clear();
    let _ = io::stdin().read_line(&mut input);

    if input.contains('1') {
        println!("HARD")
    } else {
        println!("EASY")
    }
}
